adobe/spectrum-web-components
# Wireit build

## Description
Expand `wireit` usage into the build workflow to allow for easier chaining and process cacheing.
- leverages in CI, where we get almost a 1 minute speed up
- opens the door for further simplifying our workflows, e.g. ever script that has a `:ci` could leverage `wire` it instead an remove that variant.
- even the tests themselves, in some cases, could have their results cached allow us not to run them in noop changes.

## Motivation and context
Let's go faster!

## How has this been tested?
    1. Take this branch onto your local system
    2. Run `yarn` a couple of times.
    3. See cache filled runs take ~5s, as opposed to 40-70s otherwise
    1. [View this branch of CI](https://app.circleci.com/pipelines/github/adobe/spectrum-web-components?branch=wireit-build)
    2. See that its runs are consistently faster that `main`
    3. See that its steps complete and deliver all of the information about the job that you've come to expect with no errors or warning.

## Screenshots (if appropriate)
<img width="280" alt="image" src="https://user-images.githubusercontent.com/1156657/229508249-8f4dc453-a463-4312-b5eb-25b72ea7f2b9.png">

## Types of changes

## Checklist
